spec/shared_stripe_examples/subscription_examples.rb

```diff
@@ -117,6 +117,38 @@
       expect(subscriptions.data.first.metadata.example).to eq( "yes" )
     end
 
+    it "adds a new subscription with on_behalf_of set" do
+      plan
+      customer = Stripe::Customer.create(source: gen_card_tk)
+      subscriptions = Stripe::Subscription.list(customer: customer.id)
+
+      expect(subscriptions.data).to be_empty
+      expect(subscriptions.count).to eq(0)
+
+      sub = Stripe::Subscription.create({ plan: 'silver', customer: customer, on_behalf_of: "acc_3483821", metadata: { foo: "bar", example: "yes" } })
```

Comment:
I see what you're testing here, but since Stripe requires item prices to create a subscription, should spec cases reflect this need?  Looking at https://stripe.com/docs/api/subscriptions/create , but maybe it's not critical here?

Reply:
It's not, I believe this is already tested with the PR we merged about prices here.